src/main/java/org/ambraproject/wombat/service/PeerReviewServiceImpl.java

```diff
@@ -134,11 +144,34 @@ String getAllReviewsAsXml(List<Map<String, ?>> reviewLetterItems) throws IOExcep
     // append the acceptance letter
     peerReviewContent += acceptanceLetter;
 
+    peerReviewContent += "<received-date>" + articleMetadata.get("receivedDate") + "</received-date>";
+
     // wrap it in a root node
     peerReviewContent = "<peer-review>" + peerReviewContent + "</peer-review>";
     return peerReviewContent;
   }
 
+   /**
+   * Get article metadata to include in peer review history, such as the
+   * received date (aka, the original submission date).
+   *
+   * @param itemTable
+   * @return map with article metadata. 
+   */
+  Map<String,String> getArticleMetadata(Map<String,?> itemTable) throws IOException {
```

Comment:
This name could confuse, given that there already exists an `ArticleMetadata` class that gets metadata from the rhino DB. Maybe call it `getArticleXmlData` or just get the date since that's all you really need.

Reply:
I've simplified to just fetching received date which may be all we ever need from article (YAGNI, as you said).